en/file_formats/geofence.md

```diff
@@ -0,0 +1,80 @@
+# GeoFence Plan File Format (Mission Protocol)
+
+GeoFence plan files are formatted using JSON, and can defined a cyclindrical geofence using parameters <!-- (autopilot-specific?) --> and/or an arbitrary polygon.
+
+<!--  doc originates from *QGroundControl Dev Guide*: https://dev.qgroundcontrol.com/en/file_formats/fence.html -->
+
+```
+{
+    "fileType": "GeoFence",
+    "groundStation": "QGroundControl",
+    "parameters": [
+        {
+            "compId": 1,
+            "name": "FENCE_ENABLE",
+            "value": 1
+        },
+        {
+            "compId": 1,
+            "name": "FENCE_TYPE",
+            "value": 4
+        },
+        {
+            "compId": 1,
+            "name": "FENCE_ACTION",
+            "value": 0
+        },
+        {
+            "compId": 1,
+            "name": "FENCE_ALT_MAX",
+            "value": 0
+        },
+        {
+            "compId": 1,
+            "name": "FENCE_RADIUS",
+            "value": 0
+        },
+        {
+            "compId": 1,
+            "name": "FENCE_MARGIN",
+            "value": 0
+        }
+    ],
+    "polygon": [
+        [
+            47.634457973002796,
```

Comment:
Are these ordered? Or are they arranged automatically such that there is no cross-over of the borders?